ivy/functional/frontends/numpy/statistics/histograms.py

```diff
@@ -0,0 +1,29 @@
+import ivy
+from ivy.functional.frontends.numpy.func_wrapper import (
+    to_ivy_arrays_and_back,
+    from_zero_dim_arrays_to_scalar,
+)
+
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def histogram(
+    a,
+    *,
+    bins=10,
+    range=None,
+    density=None,
+    weights=None,
+):
+    [x.dtype for x in [a, weights]]
```

Comment:
as the ivy histogram function are able to handle the default values from numpy, could you please try to reduce additional codes here?
unless there are necessary for the tests to pass, please try to minimise additional code!